.github/workflows/tests.yml

```
@@ -16,7 +16,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: '22'
```

Slightly better:

```suggestion
          node-version-file: '.node-version'
```

And put `22.15.0` in `.node-version` in repo root

---

hm, can we use it in package.json too?

---

Tools like `fnm` can read the `engines` field from `package.json` (fun fact, I added support for it in https://github.com/Schniz/fnm/pull/839) but `setup-node` cannot.